funnel/templates/scan_badge.html.jinja2

```diff
@@ -0,0 +1,143 @@
+{% endblock %}
+
+{% block content %}
+  <div class="row" id="scan-badge-wrapper">
+    {% raw %}
+    <script id='scan-badge-template' type='text/ractive'>
+
+      <div class="col-lg-4 col-xs-12">
+        <h1>{{ spaceTitle }}</h1>
+        <h3>{{ eventTitle }}</h3>
+        <h5>Please scan your badge to check-in</h5>
+        {{#if cameras}}
+          <h5>Cameras</h5>
+          {{#cameras:camera}}
+            {{#if activeCameraId === cameras[camera].id}}
+              {{#if !qrReaderOn}}
+              <button on-click="startCamera(cameras[camera])" class="btn btn-primary btn-sm btn-block">Start {{ formatName(cameras[camera].name) }}</button>
+              {{/if}}
+            {{else}}
+              <button on-click="startCamera(cameras[camera])" class="btn btn-primary btn-sm btn-block">Start {{ formatName(cameras[camera].name) }}</button>
+            {{/if}}
+          {{/}}
+        {{else}}
+          <p>No camera found</p>
+        {{/if}}
+      </div>
+
+      <div class="col-lg-8 col-xs-12">
+        <video id="reader" class="qrcode-reader"></video>
+      </div>
+
+      <div class="modal fade" id="status-msg" tabindex="-1" role="dialog">
+        <div class="modal-dialog" role="document">
+          <div class="modal-content">
+            <div class="modal-header {{#if !scanning && !attendeeFound}}bg-danger{{/if}}">
+              <button type="button" class="close" data-dismiss="modal" aria-label="Close"><span aria-hidden="true">&times;</span></button>
+              <h4 class="modal-title">Check in status</h4>
+            </div>
+            <div class="modal-body">
+              {{#if scanning}}
+                <h4>Scanning <i class="fa fa-spinner fa-spin fa-2x"></i></h4>
+              {{elseif attendeeFound}}
+                <h4>Hello {{attendeeName}},</h4>
+                <p>Thank you for choosing to participate in {{ spaceTitle }} and for being a part of this growing community.</p>
+                <p>HasGeek</p>
+              {{else}}
+                <h4>Attendee details not found</h4>
+              {{/if}}
+            </div>
+            <div class="modal-footer">
+              <button type="button" class="btn btn-default" data-dismiss="modal">Close</button>
+            </div>
+          </div>
+        </div>
+      </div>
+
+    </script>
+    {% endraw %}
+  </div>
+{% endblock %}
+
+{% block footerscripts %}
+  <script type="text/javascript" src="https://cdnjs.cloudflare.com/ajax/libs/webrtc-adapter/3.3.3/adapter.min.js"></script>
+  <script type="text/javascript" src="https://rawgit.com/schmich/instascan-builds/master/instascan.min.js"></script>
+  <script>
+    $(document).ready(function() {
+
+      var checkinApiUrl = "{{ url_for('checkin_puk', profile=space.profile.name, space=space.name, name=event.name) }}"
+
+      window.scanBadge = new Ractive({
+        el: '#scan-badge-wrapper',
+        template: '#scan-badge-template',
+        data: {
+          spaceTitle: "{{ space.title }}",
```

Comment:
See if you write all the server-side values to a hash map, and have this line read from that? It's better separation, and will let you do updates more seamlessly.